mtbf_driver/tests/mtbf/camera/test_mtbf_camera_multiple_shots.py

```diff
@@ -19,6 +19,7 @@ def test_capture_multiple_shots(self):
         self.previous_number_of_pictures = len(self.data_layer.picture_files)
 
         self.camera = Camera(self.marionette)
+        self.camera.kill_abnormal_camera_app()
         self.camera.launch()
```

Comment:
Yes! it will...if the app is already launched, the app should not restart again...it seems no other impacts on the following steps....should I tap the home button if the validation is passed.